templates/tab/ts/default/src/components/sample/AzureFunctions.tsx

```diff
@@ -17,25 +17,30 @@ async function callFunction() {
       },
     });
     return response.data;
-  } catch (err) {
-    let funcErrorMsg = "";
-    if (err?.response?.status === 404) {
-      funcErrorMsg = `There may be a problem with the deployment of Azure Function App, please deploy Azure Function (Run command palette "TeamsFx - Deploy Package") first before running this App`;
-    } else if (err.message === "Network Error") {
-      funcErrorMsg =
-        "Cannot call Azure Function due to network error, please check your network connection status and ";
-      if (err.config.url.indexOf("localhost") >= 0) {
-        funcErrorMsg += `make sure to start Azure Function locally (Run "npm run start" command inside api folder from terminal) first before running this App`;
+  } catch (err: unknown) {
+    if (axios.default.isAxiosError(err)) {
```

Comment:
NIT: prefer to use early return to reduce indent size

Reply:
I also considered early return, but I think by convention, we put the unhandled case at last of catch block:
```
catch (err: unknown) {
  if (err instanceof SomeError) {
    ...
  } else if (err instanceof AnotherError) {
    ...
  } else {
    throw err;
  }
}
```